dev-client/scripts/localization/transform-to.mjs

```diff
@@ -77,35 +77,26 @@ const transform = (process, from, i18Transform) => {
 // PO transform
 const toPoOptions = {
   project: 'Terraso',
+  ctxSeparator: '',
```

Comment:
Question: This is saying that we will never use the concept of context, right?

Reply:
YEs. If we do, we will have to come up with a different separator (default is `_`) and update this configuration.